DigitalLearningSolutions.Data/Models/TrackingSystem/DateInformation.cs

```diff
@@ -18,7 +18,7 @@ public string GetDateLabel(string format)
         {
             var quarter = Date.Month / 3 + 1;
 
-            format = format.Replace("Q", $"Quarter {quarter}").Replace("q", $"Q{quarter}");
+            format = format.Replace("Q", $"'Quarter' {quarter}").Replace("q", $"Q{quarter}");

```

Comment:
Since these strings are getting put into DateTime.ToString(), we have to escape any of the strings found here: https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings

If we don't to this, they'll get replaced with the DateTime parts that it matches. In this case the t in Quarter gets replaced with A or P depending on if the time is AM or PM. Similarly the "Week commencing" was getting replaced with "Week co00encinA.D." because the mm matches minutes, and g is the period (AD/BC/CE/BCE e.t.c. depending on current culture settings)

Reply:
Ah I see